CONTRIBUTING.md

```diff
@@ -1,39 +1,87 @@
 # Contributing
 
-Hi there! We're excited you've got ideas for an existing or new topic page. Your help makes Explore valuable to others in your community.
+Hi there! We're excited you've got ideas to improve topics and collections. Your helping the community discover valuable information.
 
 This project adheres to the [Contributor Covenant Code of Conduct](CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code.
 
-There are a few ways you can contribute: improving an existing topic page, or curating a new topic page.
+There are a few ways you can contribute:
+
+- Improving an existing topic or collection
+- Curating a new topic topic or collection
 
 As you write content, check out the [Style Guide](./docs/styleguide.md) to learn what each field means, and how they should be formatted. Following the style guide will improve the chances of your contribution being accepted.
 
-Note: updates won't immediately appear once we've merged your PR. We pull in these changes regularly to GitHub.
+Note: Updates won't immediately appear once we've merged your PR. We pull in these changes regularly to GitHub.
 
-## Improving an existing topic page
+## Improving an existing topic or collection
 
-If a topic page already exists, it will be listed in the [topics directory](https://github.com/github/explore/tree/master/topics). The topic name should match its URL, e.g. `https://github.com/topics/rails` corresponds to the [`rails` folder](https://github.com/github/explore/tree/master/topics/rails).
+If a topic or collection already exists, it will be listed in its respective directory:
 
-To improve an existing topic page, please **open a pull request** with your proposed changes:
+- [Topics](https://github.com/github/explore/tree/master/topics)
```

Comment:
How would you feel about making these appear more directory-ish? Eg:

> If a topic or collection already exists, it will be listed in its respective directory:
> 
> - [`topics/`](https://.com/github/explore/tree/master/topics)
> - [`collections/`](https://github.com/github/explore/tree/master/collections)